flowzone.yml

```diff
@@ -1366,10 +1369,13 @@ jobs:
           comment="$(gh pr view ${{ github.event.pull_request.number }} --json comments \
             --jq '.[].[] | select(.author.login=="${{ needs.versioned_source.outputs.author }}").body')"
 
-          EOF="$(openssl rand -hex 16)"
-          echo "text<<$EOF" >> $GITHUB_OUTPUT
-          echo "${comment}" >> $GITHUB_OUTPUT
-          echo "$EOF" >> $GITHUB_OUTPUT
+          # send message only if a human processed the stubbed release notes, otherwise no-op
+          if ! [[ "$comment" =~ '[only keep the important and rephrase]' ]]; then
```

Comment:
@ab77 
 As I'm lacking context, how is this making sure that a human reviewed the release notes?

Reply:
well, the assumption is that if the boiler plate text is still there, no one has reviewed